docker/ansible-in-docker/build.bash

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -vx
+
+function buildAnsible {
+    dockerTag=oso-ansible:${ansibleVersion}
+
+    bashScript=./oso-ansible-playbook-${ansibleVersion}
+
+    docker build --tag ${dockerTag} - << EndOfDockerfile
+FROM fedora:25
+
+RUN yum -y update \
+ && yum -y install ${ansibleRPM} \
+ && yum clean all
+EndOfDockerfile
+
+    cat > ${bashScript} << EndOfBashScript
+#!/bin/bash
+#set -vx
+
+sudo docker run --rm -it --privileged \\
+  -v /etc/ansible:/etc/ansible:ro \\
+  -v /usr/share/ansible/inventory/multi_inventory.py:/usr/share/ansible/inventory/multi_inventory.py:ro \\
+  -v /:/host:ro \\
```

Comment:
As the container is short-lived (life-of-script, as opposed to host-monitoring-daemon), I'm not 100% sure this will be a problem. I'll try add it.